### ISSUE TITLE
Critère 3.2 : valeur en pixel de la taille de police agrandie en gras

### ISSUE THREAD

**Issue by inseo** [accessibility-luxembourg/ReferentielAccessibiliteWeb#2]
Dans la suite de tests du critère 3.2, la valeur de 18,5px est retenue pour la taille de police agrandie en gras.
Or si l'on se réfère aux WCAG, [la valeur exacte est 14pt](https://www.w3.org/TR/WCAG22/#dfn-large-scale).

D'après le [tableau de correspondance](https://www.w3.org/TR/css-values/#pt) qui est fourni :   
- `1 pt` équivaut à `1/72 in`
- `1 px` équivaut à `1/96 in`

Partant de là : `14 pt` donnent `0,1944444444 in` soit `18,6666666667 px`.
Considérer `18.5px` comme la valeur de taille agrandie pour du texte en gras est donc inexacte.

Accessoirement, ça expliquerait pourquoi certains outils comme WAVE ou l'inspecteur d'accessibilité des outils de développement de Firefox jugent tous deux le contraste de l'intitulé "Aide-soignants" insuffisant sur [cette page](https://healthcareers.public.lu/fr/a-propos.html) malgré une taille de police de 18.5px en gras pour un rapport de contraste de 3.34:1.

---

PS : Tout le crédit revient à [Jeanne Lazzarelli](https://github.com/jeanne-l) pour avoir identifier cette divergence. 👍

**Comment by AlainVagner** [comment]
Effectivement, on pourrait passer ici à 18.7px. Je pense qu'on peut arrondir à 18,7px et non pas utiliser 10 chiffres après la virgule si ça va.

C'est ce qui est recommandé notamment dans le [BITV](https://bitvtest.de/pruefschritt/bitv-20-web/bitv-20-web-9-1-4-3-kontraste-von-texten-ausreichend).

**Comment by AlainVagner** [comment]
Je réouvre cette issue. On est bien d'accord qu'il s'agit d'un arrondi et de savoir quel arrondi prendre en compte.
Dans l'[understanding de ce critère WCAG](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum) on retrouve la mention suivante :  "The ratio between sizes in points and CSS pixels is 1pt = 1.333px, therefore 14pt and 18pt are equivalent to approximately 18.5px and 24px."
WCAG mentionne donc cette approximation à 18.5px. 
D'autre part, certains outils semblent utiliser d'autres approximations, par exemple l'outil WCAG Colour contrast checker utilise une valeur de 19px.
Dans ces conditions il est difficile de passer à 18.7, la seule mention dans le texte du W3C étant 18.5.
Je laisse ce ticket ouvert pour qu'on puisse continuer la discussion.

**Comment by AlainVagner** [comment]
Si la valeur exacte est à 18.666666666... px on peut argumenter que 18.5 px est une approximation insuffisante, dans la mesure où on va demander à des textes en gras à 18.6 px un contraste de 3:1 au lieu de 4.5:1. L'arrondi à choisir devrait être supérieur à la valeur exacte.

**Comment by juliemoynatPro** [comment]
Point très intéressant relevé. Puisque WCAG mentionne "18.5px", je suis donc allée fouiller du côté du GitHub des WCAG et je suis tombée sur une discussion de 2016 qui a déviée sur exactement le même point "14pt = 18.666px" : https://github.com/w3c/wcag/issues/181#issuecomment-212946321
Il semble que le choix a été fait sciemment.

J'aime bien ton dernier argument, @AlainVagner.

Toutefois, j'aurais tendance à dire que, si la valeur doit être remise en question, ce devrait être d'abord du côté des WCAG en faisant un ticket précisément sur ce sujet avec pour but d'harmoniser la pratique.

**Comment by goetsu** [comment]
https://github.com/w3c/wcag/issues/4337

**Comment by AlainVagner** [comment]
Merci @goetsu pour le ticket ouvert sur le repo de WCAG. À voir si la discussion va plus loin que dans le ticket mentionné par @juliemoynatPro.
